src/phases/execution/ClusteredCollection.yml

```
@@ -73,25 +75,34 @@ Actors:
             key: {_id: "hashed"}
             numInitialChunks: 12
 
-# Phase 2: insert. TODO Make multithreaded once TIG-2938 is resolved.
 - Name: Insert
   Type: Loader
-  Threads: 1
+  Threads: *LoadingThreads
   Phases:
     OnlyActiveInPhases:
       Active: [2]
       NopInPhasesUpTo: *MaxPhases
       PhaseConfig:
         Repeat: 1
         Database: *Database
-        Threads: 1
+        MultipleThreadsPerCollection: true
         CollectionCount: 1
         DocumentCount: *docCount
         BatchSize: 1000
         Document:
           # Start from {_id: "0000001"} in order to be able to set the ^RandomInt max bound to *docCount
           # in the LookupByClusterKey phase. This works around TIG-3759.
-          _id: {^Join: {array: [{ ^FormatString: {"format": "%07d", "withArgs": [{^Inc: {start: 1}}]}}, *PaddingGenerator]}}
+          # The multiplier uses the actor ID so we start at -3*DocsPerThread + 1 to get to 1 as the
+          # first _id.
```

If I'm interpreting the below and [this thread](https://github.com/mongodb/genny/pull/941#discussion_r1253616420) correctly, the `multiplier` is an offset that is added to `start` and is computed as the `multiplier` value multiplied by the phase id, where the current phase is phase 3.

If the above is accurate, I suggest:
```suggestion
          # 'multiplier' is an offset to 'start', and the 'multiplier' value itself is multiplied by the phase id (3).  So we start at -3*DocsPerThread + 1 to get to 1 as the first _id.
```
And I also suggest refreshing the ["Phase" comments](https://github.com/mongodb/genny/pull/941/files#diff-7d05c7e5cc91b2536c056d4dde59005a6d5a1fe8320ca2715abe6d203c9ef678R36) that seem to be off by 1.

---

I can rephase the comment as you suggest but it's not quite right to say phase ID. It's by actor ID not phase ID. The idea is that each thread is a separate actor, so we need actor ID not phase ID to ensure they don't create duplicate ID values.

I think @naama-bareket mentioned 'phase 3' just to say that "there are at least 2 other actors that went before this", and often the setup phases are a single actor, so the two are sometimes interchangeable.